samples/NovelRT.DotNet.Sample/NovelRT.DotNet.Sample.csproj

```diff
@@ -31,4 +22,9 @@
     <Message Importance="High" Text="@(ResolvedFileToPublish->'$(PublishDir)%(RelativePath)')" />
   </Target>
 
+    <!-- DO NOT REMOVE: Used to ensure the project is only built via CMake -->
```

Comment:
Maybe we should export these to a Directory.Build.targets :thinking:

Reply:
I can do that when I add the tests project. Will also let us normalize some other settings and block automatic import of any other Dir.Build stuff in the file tree